data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3482 @@
+
+
+
+mission "Unfettered: Attracted Attention"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove your strength 2: active"
+	on offer
+		require "Jump Drive" 1
+	on enter "Almaaz"
+		dialog `The Unfettered comes to the bridge to warn you, "There are even more unfettered humans than anticipated. I told you not to come here with a jump drive!"`
+	npc evade
+		government "Pirate"
+		system "Almaaz"
+		personality disables plunders vindictive
+		fleet "Large Northern Pirates"
+
+mission "Unfettered: Jump Drive Retrieval"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove Your Strength 2: active"
+	on offer
+		conversation
+			`Following the advice of the Hai, you park your ship close to the hidden depot, yet still close enough to the spaceport as to not draw unwanted attention by landing off grid. Your crew should be able to intervene should anything go wrong. Then you follow him to the hatch. He is moving at a quick pace, you have difficulties following him, even given the fact he is wearing what seems to be an heavy combat suit.`
+			choice
+				`	"Who made this vault, what should I expect?"`
+				`	"Should we not take more men to ensure our safety?"`
+					goto moremen
+				`	(No time for talking.)`
+					goto outsideship
+			`	"That is an information I cannot trust anyone with, not even you, and especially not here. I do not want the owner of this cache to be in danger whilst we need jump drives to expand. Assume it is owned by unfettered humans, or pirates as I think you call them, and that we are ready to face them," he tells you.`
+				goto outside
+			label moremen
+			`	"We are trying to be discrete here, not to draw attention! The more we are the more we will attract to attack us, we want them to think we have no value, not that we are rich fellows treasure hunting." Judging from his tone, this was a stupid question.`
+			
+			label outsideship
+			`	Before exiting, he warns you, "Whatever happens, do not try anything stupid, and stay behind me at all times." He seems worried and keeps looking around furtively, making as little sound as possible with his suit. He is impressively stealthy though, and you surmise the suit must have been crafted specifically for him in order to be so discrete. He does not have his helmet on as he peers cautiously around corners and assess his surroundings.`
+			choice
+				`	"Is everything alright?"`
+				`	"There is no need to be worried, that cache is well hidden."`
+				`	(Keep walking, ready to reach for my gun.)`
+					goto discrete
+			`	You are not sure what kind of sign he just did, but it seems pretty clear he wants you to keep quiet. Looking around, you notice the two of you have attracted quite a bit of attention from the locals. While they all seem to have nothing to lose, the vast majority of them are very young and probably not the hardened criminals you had expected.`
```

Comment:
What is your setting at this point? Have you arrived somewhere in a town? On the outskirts? A natural feature near an outlying village?
Where you are and what the setting it is important for anything in this paragraph to make sense.

Also the locals make precisely 0 sense leading into the next paragraph, so you need to either clarify the transition, or get rid of them entirely.

Reply:
I think we could go with saying that some of those the players assumes to be "locals" are in fact just part of the ones who took control of the vault, disguised as them. There are indeed no "locals" on sight when you approach the vault, they are hidden.
I would need to suggest that somehow though, saying that something is wrong; some seem too old for this?

Reply:
I clarified the setting, and added later a suspicious individual dressed as a local to trigger the senses of the player.
Its spaceport => small mountain => village => the mountain of destination